README.md

```diff
@@ -1,2 +1,321 @@
 # std-toast
 This document scopes out a web platform API for a 'toast' pop-up notification.
+
+## Why?
+
+Modern web applications allow users to complete lots of actions per page,
+which necessitates providing clear feedback for each action.
+Toast notifications are commonly used to unobtrusively provide this feedback.
+
+Many libraries in a variety of frameworks implement a version of toast
+(see [research](./study-group/)),
+but the web has no built-in API to address the use case.
+By providing a toast API as part of the web platform's standard library,
+the web becomes more competitive with other app development platforms,
+and web application developers can spend less of their time and bytes
+on implementing this pattern.
+
+Toasts are also a deceptively-tricky pattern to get right.
+They require special accessibility considerations,
+and scenarios involving multiple toasts need special handling.
+[Not all libraries account for these subtleties](./study-group/).
+By providing a built-in toast control that fully handles these aspects,
+we can level up the typical toast experience for both developers and users of the web.
+
+Finally,
+the ecosystem can benefit from a shared understanding of how to create and style toasts.
+If the platform provides a toast,
+then all libraries and components can freely use toasts to communicate to their users.
+Whereas,
+if toasts can only be found in libraries,
+then importing a toast-using component also means importing their opinion on what the best toast library is.
+In the worst case,
+this can lead to multiple uncoordinated toast libraries acting on a single page,
+each of which needs its own styling and tweaks to fit in to the application.
+If instead libraries and components all use the standard toast,
+the application developer can centrally style and coordinate them.
+
+
+## Sample code
+
+The standard toast can be used according to two different patterns.
+
+The first defines a `<std-toast>` HTML element,
+then shows it with configurations via a method on the element.
+This can be used to declaratively predefine toasts the application will need,
+and then show them inside the application logic.
+
+```html
+<script type="module">
+import 'std:elements/toast';
+</script>
+
+<std-toast id="sample-toast" theme="info">
+    Hello World!
+</std-toast>
+```
+
+```js
+document.querySelector('#sample-toast').show({
+    duration: 3000
+});
+```
+
+The second imports the `showToast()` function from the `"std:elements/toast"` module,
+which takes in a message and some configurations and creates and shows a toast in the DOM.
+This is more convenient for one-off toasts,
+or for JavaScript-driven situations,
+similar to how `alert()` is currently used.
+
+```js
+import { showToast } from 'std:elements/toast';
+
+const toast = showToast("Hello World!", {
+    theme: "info",
+    duration: 3000
+});
+```
+
+## Goals
```

Comment:
We should add an explicit goal to provide the foundation around which other toast libraries can layer themselves, and provide more opinionated defaults, specific styles, or better framework integration. Maybe with a "TODO: link to an example".

Reply:
That seems like a good addition to me, in keeping with explaining the spirit of why we're building this.